tracer/test/snapshots/AspNetMvc4Tests.Integrated.NoFF.__path=_Home_BadRequestWithStatusCode-statuscode=401&TransferRequest=true_statusCode=401.verified.txt

```diff
@@ -55,11 +59,15 @@
 System.Exception: Oops, it broke. Specified status code was: 401
 at Samples.AspNetMvc4.Controllers.HomeController.BadRequestWithStatusCode(Int32 statuscode),
       error.type: System.Exception,
+      http.client_ip: 127.0.0.1,
       http.method: GET,
       http.request.headers.host: localhost:00000,
+      http.route: {controller}/{action}/{id},
```

Comment:
TODO: Add a test case for TransferRequest to make sure it works properly. I think it's only successful right now because both of the `aspnet-mvc.request` spans have the same http.route value. I would hazard a guess that it's the *last* http.route calculation that wins, where instead we would only want the route from the first HTTP request of the TransferRequest scenario to be placed on the root span

Reply:
Resolving because this can be addressed in [this comment](https://github.com/DataDog/dd-trace-dotnet/pull/2915/files#r922536272)

Reply:
I'm not sure which comment as the page refuses to navigate, this one above you meant? 😅 

> I think I understand what's going on. I think we're entering AspNetResourceNameHelper.CalculateResourceName...

Reply:
😅 Sorry about that, this should be taken care of by the other TransferRequest comment I made

Reply:
should be fixed by 4d2355566ff23e3458e03b9e8a67f86b3c158b29 , not setting it a second time if it already exists in the `aspnetmvcintegration`